sqlite.go

```diff
@@ -1312,6 +1442,14 @@ func newDrv() *sqlite { return &sqlite{} }
 // unnecessary; the sql package maintains a pool of idle connections for
 // efficient re-use.
 //
+//
+// we adds the following query parameters to those used by SQLite:
+//   _mattn=true
```

Comment:
@mattn Can you please let us know if you don't mind using your handle this way? We can use some other string if you prefer. Thank you.